nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -2580,7 +2580,9 @@ public void removeControllerService(final ControllerServiceNode service) {
                 throw new IllegalStateException("ControllerService " + service.getIdentifier() + " is not a member of this Process Group");
             }
 
-            service.verifyCanDelete();
+            if (!service.isMoving()) {
```

Comment:
Is there a reason for not checking moving status inside the `verifyCanDelete` method?